EIPS/eip-7923.md

```diff
@@ -55,7 +63,7 @@ A transaction-global memory limit is imposed. If the number of pages allocated i
 
 ## Rationale
 
-Benchmarks were performed on a 2019-era CPU, with the ability to keccak256 around 256MB/s, giving it a gas-to-ns ratio of 20 ns per 1 gas. The following benchmarks were performed:
+Benchmarks were performed on a 2019-era CPU, with the ability to `keccak256` around 256MB/s, giving it a gas-to-ns ratio of 20 ns per 1 gas (given that `keccak256` costs 6 gas per word). The following benchmarks were performed:
```

Comment:
I would clarify here that a word is 32 bytes. This has lead (for me) to a lot of confusion and also helps te reader to quickly re-calculate this result. (Otherwise the 256/32 division step might be forgotten)

Reply:
https://github.com/ethereum/EIPs/pull/9635/commits/defacc1e2b61bb68e66432d20af3241758cb5378